pepper-apis/src/test/java/org/broadinstitute/ddp/route/DsmRouteTest.java

```diff
@@ -160,9 +161,18 @@ private static CachedDsmCredentials tryCachedDsmCredentials(String auth0Domain,
             return null;
         }
 
-        DecodedJWT jwt = JWTConverter.verifyDDPToken(creds.getToken(), JWTConverter.defaultProvider(auth0Domain));
-        if (jwt == null) {
-            LOG.warn("Unable to verify or decode jwt token for cached dsm credentials");
+        DecodedJWT jwt;
+        try {
+            jwt = JWTConverter.verifyDDPToken(creds.getToken(), JWTConverter.defaultProvider(auth0Domain));
+            if (jwt == null) {
+                LOG.warn("Unable to verify or decode jwt token for cached dsm credentials");
+                return null;
+            }
+        } catch (TokenExpiredException e) {
+            LOG.warn("Cached dsm token expired, not using", e);
```

Comment:
When you run tests locally and if the token expired it will fail the tests. Fixed it by catching that exception here. This one is for the DSM token used for DSM routes.